lib/client/api.go

```diff
@@ -231,6 +231,9 @@ type Config struct {
 
 	// KeyTTL is a time to live for the temporary SSH keypair to remain valid:
 	KeyTTL time.Duration
+	// CliTTL is used to ensure the TTL passed by flag is used over
+	// the cluster default
+	CliTTL bool
```

Comment:
I'm not sure about this. If I'm reading this write, does this allow the CLI to specify a TTL greater than the cluster's max session TTL?